lib/info-extras.js

```diff
@@ -124,11 +124,37 @@ exports.getPublished = (body) => {
  * @return {Array.<Object>}
  */
 exports.getRelatedVideos = (body) => {
-  let jsonStr = util.between(body, '\'RELATED_PLAYER_ARGS\': {"rvs":', '},');
+  let jsonStr = util.between(body, '\'RELATED_PLAYER_ARGS\': ', '\n');
+  let watchNextJson;
   try {
-    jsonStr = JSON.parse(jsonStr);
-  } catch (err) {
-    return [];
+    jsonStr = jsonStr.substring(0, jsonStr.length - 1)
+    jsonStr = JSON.parse(jsonStr)
+    watchNextJson = JSON.parse(jsonStr["watch_next_response"])
```

Comment:
use the dot syntax when you can for non-dynamic properties
```suggestion
    watchNextJson = JSON.parse(jsonStr.watch_next_response)
```